crates/sncast/tests/data/multicall_configs/deploy_invoke_calldata_ids.toml

```diff
@@ -1,19 +1,19 @@
 [[call]]
 call_type = "deploy"
-class_hash = "0x7644BE7F58307726AA836E945EDEDE13E9E08C38EAF2186D4D48ECA7DD435AC"
+class_hash = "0x7644be7f58307726aa836e945edede13e9e08c38eaf2186d4d48eca7dd435ac"
 inputs = []
 id = "map_contract"
 unique = false
 
 [[call]]
 call_type = "invoke"
-contract_address = "0x50E0FDC4FEEB0CB85F175D40EC2F8D622A92CCAC3E59419C773B4EAAD2B5A2"
+contract_address = "0x50e0fdc4feeb0cb85f175d40ec2f8d622a92ccac3e59419c773b4eaad2b5a2"
 function = "put"
 inputs = ["0x123", "map_contract"]
 
 [[call]]
 call_type = "deploy"
-class_hash = "0x5CADC0ECBC6E2A502A7A7E8E5B55400A1A92AFCCB68ED5DC5EFAD86C1FC4EDB"
+class_hash = "0x6e10d493d7c807e0fbaad4f0c31792f24d64747fa328830a68cb5d2313f9a55"
```

Comment:
why did this change the value?

Reply:
The issue was left over, cached artifacts. This is something Igor is trying to fix in an issue he's working on

Reply:
I have resolved the conflicts :+1: